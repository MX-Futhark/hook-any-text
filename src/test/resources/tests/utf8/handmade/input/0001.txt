48692074686572652ee3808054686973206973206120746573742e0ae3818ae381afe38288e38186e38194e38196e38184e381bee38199e38082e38386e382b9e38388e381a7e38199e38082