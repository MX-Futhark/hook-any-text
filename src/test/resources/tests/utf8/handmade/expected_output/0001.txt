["Hi there.　This is a test.
おはようございます。テストです。"]